qpc/report/insights.py

```diff
@@ -115,8 +120,13 @@ def _validate_args(self):
             )
 
     def _handle_response_success(self):
+        file_content = None
+        if self.args.path:
+            file_content = self.response.content
+        else:
+            file_content = extract_json_from_tar(self.response.content)
         try:
-            write_file(self.args.path, self.response.content, binary=True)
+            write_file(self.args.path, file_content, binary=True)
             print(_(messages.REPORT_SUCCESSFULLY_WRITTEN))
```

Comment:
a line like this one seems problematic when writing output to stdout (if we were trying to pipe output to jq or any other json parser, for instance, the extra bit of info would compromise the output).

We should be following the UNIX way for all logs: log goes to stderr. output to stdout.

Reply:
I bet if you collect all output from `qpc report deployment/details/insights` as json and try to parse with json.loads it won't be able to do it.